cs251tk/toolkit/process_student.py

```diff
@@ -24,16 +25,26 @@ def process_student(
 ):
     if clean:
         remove(student)
-
+    if debug:
+        logging.debug("Cloning    {}'s repository".format(student))
```

Comment:
First thing to note: when you use the "logging" module, you don't need to check "if debug" at each callsite. You can just throw stuff into the debug log, and it won't be printed unless debugging is active.

Reply:
Good to know. I'll fix that.